financial-connections-lite/src/main/java/com/stripe/android/financialconnections/lite/FinancialConnectionsSheetLiteActivity.kt

```diff
@@ -60,6 +63,26 @@ internal class FinancialConnectionsSheetLiteActivity : ComponentActivity(R.layou
         }
     }
 
+    private fun setupBackButtonHandling() {
+        onBackPressedDispatcher.addCallback(this, object : OnBackPressedCallback(true) {
+            override fun handleOnBackPressed() {
+                val exitDialog = AlertDialog
+                    .Builder(this@FinancialConnectionsSheetLiteActivity)
+                    .setTitle(R.string.stripe_fc_lite_exit_title)
+                    .setMessage(R.string.stripe_fc_lite_exit_message)
+                    .setCancelable(false)
```

Comment:
Nit: Let's make this cancelable, which will have the same effect as pressing `Cancel`.

Reply:
https://github.com/stripe/stripe-android/pull/10683/commits/f3c08f088f5ccf82e86a487e57252f8a864c970c